code/drasil-build/lib/Build/Drasil/Make/AST.hs

```diff
@@ -21,6 +21,9 @@ data CommandOpts =
 data Type = Abstract
           | File deriving Eq
 
+
+-- | A makefile Comment is a String
+type MkComment = String
```

Comment:
The reason I went with `MkComment` instead of `Comment` is because `Comment` as a type already exists in `GOOL` and I felt that I should not be importing types from `GOOL` for `drasil-build`.

https://github.com/JacquesCarette/Drasil/blob/afb6fb752b8364d2807ced7fc0c1dd6c6aba52b2/code/drasil-gool/lib/GOOL/Drasil/ClassInterface.hs#L476

I am willing to change that if you would like!

Reply:
I don't really think that there is a semantic difference between comments in code and comments in Makefiles. Do you have any thoughts @JacquesCarette @balacij?

Reply:
Sounds like a good idea to me!

Reply:
Ok, I will make that change as well